contracts/storagetrie/StorageTrieStoreTest.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity 0.4.24;

```

Comment:
Current official version of solidity compiler is 0.5.6.
```suggestion
pragma solidity ^0.5.6;
```

Reply:
@jeonghonah As far as I know and from the docs.klaytn.com, both versions are supported. Aren't they? https://docs.klaytn.com/smart-contract/solidity-smart-contract-language